packages/components/pie-modal/component-bundle-sizes.json

```diff
@@ -0,0 +1,5 @@
+{
+  "pie-modal/index.js": 25075,
```

Comment:
this file is auto generated by the plugin to store the bundle sizes to check against the next time the command is run

Reply:
Thanks for clarifying. No issues from me then

Reply:
Cheers! Worth nothing we don't necessarily need this file as the script will still just check if the size exceeds the threshold defined in the json config. 

This generated file is just to help understand exactly what the previous/current sizes are :)